modules/marsmediaBidAdapter.js

```diff
@@ -0,0 +1,130 @@
+import * as utils from '../src/utils';
+import {registerBidder} from '../src/adapters/bidderFactory';
+const BIDDER_CODE = 'marsmedia';
+
+function getDomain() {
+  if (!utils.inIframe()) {
+    return window.location.hostname
+  }
+  let origins = window.document.location.ancestorOrigins
+  if (origins && origins.length > 0) {
+    return origins[origins.length - 1]
+  }
+}
+
+export const spec = {
```

Comment:
Hi one reccomendation is to add the `supportedMediaTypes` param to your spec.

It looks like you only support banner so it would look something like this:

https://github.com/prebid/Prebid.js/blob/master/modules/adponeBidAdapter.js#L11

Reply:
OK, Thanks. I added it.